pkg/apis/ceph.rook.io/v1/resources.go

```diff
@@ -37,6 +37,8 @@ const (
 	ResourcesKeyCrashCollector = "crashcollector"
 	// ResourcesKeyLogCollector represents the name of resource in the CR for the log
 	ResourcesKeyLogCollector = "logcollector"
+	// ResourcesKeyRGWOpsLog represents the name of resource in the CR for the log
+	ResourcesKeyRGWOpsLog = "rgwopslog"
```

Comment:
As noted, the resources for this container should be configured in the CephObjectStore. Possibly under an ops log config heading.

Reply:
Still needs addressed in design.

Reply:
updated as
```
gateway:
  logging:
    opsLogEnabled: true
    resources:
```